ReadInSeedClimClimateData_CleanTemp&SoilMoisture_ForOSF.R

```diff
@@ -0,0 +1,608 @@
+#########################################################
+ # READ IN AND CLEAN RAW SEEDCLIM CLIMATE DATA FROM OSF #
+#########################################################
+
+### LIBRARIES
+library("lubridate")
+library("tidyverse")
+library("data.table")
+
+# Stuff 
+pn <- . %>% print(n = Inf)
+
+#### IMPORT CLIMATE DATA FOR ALL SITES ####
+# load in premade functions
+source('Functions_ReadInSeedClimClimateData.R')
```

Comment:
This file needs to be pushed

Reply:
The "Functions_ReadInSeedClimClimateData.R" file?